README.md

```diff
@@ -0,0 +1,52 @@
+# Moodle
+
+## Installation
+
+1. Fetch the moodle distribution defined in .env
+
+    ```shell
+    task dist:build
+    ```
+
+2. Startup docker
+
+    ```shell
+   task start
+   ```
+
+3. Install moodle
+
+    ```shell
+    task install
+    ```
+
+## Upgrading moodle
+
+https://docs.moodle.org/405/en/Upgrading
+
+
+## Managing moodle
+
+Customization of moodle should be done through changes in moodle/local folder.
+
+Several scripts are available to maintain the moodle installation.
+
+### Moodle customization
+
+Moodle customizations are defined in /local folder and mapped into moodle through docker volumes.
```

Comment:
Can you add a reference to what “Moodle customization” is and what can be done?

Reply:
Still trying to figure this part out. Making a separate branch for configuration.